src/engine/core/geometry.cpp

```diff
@@ -98,7 +98,7 @@ void Frustum::computePerspective(const Vec3& position,
 	Vec3 corner2 = far_center + x * far_width + y * far_height;
 
 	float size = (corner1 - corner2).length();
-	size = Math::maximum(sqrt(far_width * far_width * 4 + far_height * far_height * 4), size);
+	size = Math::maximum(std::sqrt(far_width * far_width * 4 + far_height * far_height * 4), size);
```

Comment:
without `std` `sqrt` is a C function which takes and returns double. Math::maximum takes arguments of equal types.
Besides I think sqrt should compute in floats since all computations here do that.